Cargo.toml

```diff
@@ -23,13 +25,13 @@ libc = "0.2"
 parking_lot = "0.12.2"
 paste = "1.0.14"
 raw-cpuid = "11.0.1"
-strum = { version = "0.25.0", features = ["derive"] }
+strum = { version = "0.26", features = ["derive"] }
```

Comment:
Why are we switching to no patch version?

Reply:
Why not? Do we mind if dependencies update patch levels?

Reply:
Aren't those compatible so that's still allowed?  I'd just been doing what `cargo add` defaults to, and all the other dependencies are still like that.

Reply:
The patch level would get updated with `cargo update` but we don't get updates to the next minor version (I had to manually edit this to update). I'd prefer we get minor version updates when running `cargo update`, so eventually I'd like to switch to "x.y" for all deps unless there are other issues. But that doesn't need to be done right now.

Reply:
Okay if that's what we intend to do later then this is fine now.